projects/ids-enterprise-ng/src/lib/datagrid/soho-datagrid.d.ts

```diff
@@ -799,6 +807,9 @@ interface SohoDataGridColumn {
   /** css class  */
   cssClass?: SohoDataGridColumnCssClassFunction | string;
 
+  /** disable button  */
+  disableButton?: SohoDataGridColumnDisableButtonFunction;
```

Comment:
Lets call the setting `disabled` as per other PR

Reply:
Updated now